run.cmd

```diff
@@ -30,10 +30,10 @@ if NOT [%ERRORLEVEL%]==[0] (
 set _toolRuntime=%~dp0Tools
 set _dotnet=%_toolRuntime%\dotnetcli\dotnet.exe
 
-echo Running: %_dotnet% %_toolRuntime%\run.exe %*
-call %_dotnet% %_toolRuntime%\run.exe %*
+echo Running: %_dotnet% %_toolRuntime%\run.exe %~dp0config.json %*
```

Comment:
We should port this back to buildtools, good spot to put the same path in the echo.